csharp-basic-topics/FileAccessModifierInCsharp/Tests/ContentUploadServiceTests.cs

```diff
@@ -0,0 +1,20 @@
+namespace Tests
+{
+    public class ContentUploadServiceTests
+    {
+        [Fact]
+        public void WhenUploadCalled_ThenWritesExpectedUploadMessage()
+        {
+            // Arrange
+            var testDateTime = DateTime.Now;
+            var testFileId = 1;
+            var expectedMessage = $"{testDateTime} - Upload - File ID: {testFileId}"; // \r\n for Windows line ending
```

Comment:
We don't need the comment at the end of this line. 
```suggestion
            var expectedMessage = $"{testDateTime} - Upload - File ID: {testFileId}";
```